SemanticSearchV2.md

```diff
@@ -149,7 +145,9 @@ Parameters (query/form):
 -	user_id (optional) : user id for which to look up personalized overrides. Defaults to null. 
 -	relationships_threshold : the mimimum weight for a relationship entry to be added as enrichment. Can be used to prune the result size. Defaults to 0.5. Allowed values are any number between 0 and 1.
 -	document : the binary document to parse as base64 encoded string (according to RFC 3548/4648).
--	type : the type of document either "JOB" or "RESUME".
+-	type : the type of document is either "JOB" or "RESUME".
+-	job_title (optional, only valid when type is JOB) : a pre-define job title to override the one extracted from a job document. Notice that the job title may also be cleaned internally in order to get enrichments.
+-	location (optional, , only valid when type is JOB): a pre-defined location to override the one extracted from a job document.
```

Comment:
You have two commas after 'optional'

a pre-defined location [which] override[s] the [location] extracted from [the] job document.